Windows/MainWindowMenu.cpp

```diff
@@ -175,7 +175,9 @@ namespace MainWindow {
 
 		// We only reload this initially and when a menu is actually opened.
 		if (!menuShaderInfoLoaded) {
-			ReloadAllPostShaderInfo();
+			// This is on Windows where we don't currently blacklist any vendors, or similar.
+			// TODO: Figure out how to have the GPU data while reloading the post shader info.
+			ReloadAllPostShaderInfo(nullptr);
```

Comment:
It's not great, but could probably use `PSP_CoreParameter().graphicsContext->GetDrawContext()`, plus null checks.

-[Unknown]

Reply:
If no game has been started, that's not even initialized, right? So I'm not sure it's worth the trouble.

I'll deal with this in a better way (or fix the shader to not need blacklisting...) at a later point.